packages/java/tests/spring/react-signals/frontend/views/SharedNumberSignal.tsx

```diff
@@ -11,12 +11,16 @@ export default function SharedNumberSignal() {
 
   return (
     <div>
-      <span id="sharedValue">{sharedValue}</span>
+      <span id="sharedValue">
+        <>{sharedValue}</>
```

Comment:
This change is interesting. I got the idea of it right from the `@preact/signals-react` package [README](https://github.com/preactjs/signals/tree/main/packages/react#rendering-optimizations). With the `Fragment` used here the use of a signal without a `value` is permitted